docs/the-new-architecture/use-app-template.md

```diff
@@ -3,5 +3,61 @@ id: use-app-template
 title: Using the New App Template
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+This page will help you create a new React Native app that uses the new architecture.
+
+## Installing dependencies
+
+Start by setting up your development environment as described in [Getting Started](getting-started.md), under "React Native CLI Quickstart". Stop when you reach "React Native Command Line Interface", and resume here.
+
+<h3>React Native Command Line Interface</h3>
+
+React Native has a built-in command line interface. Rather than install and manage a specific version of the CLI globally, we recommend you access the current version at runtime using `npx`, which ships with Node.js. With `npx react-native <command>`, the current stable version of the CLI will be downloaded and executed at the time the command is run.
+
+<h2>Creating a new application</h2>
+
+> If you previously installed a global `react-native-cli` package, please remove it as it may cause unexpected issues.
+
+React Native has a built-in command line interface, which you can use to generate a new project. You can access it without installing anything globally using `npx`, which ships with Node.js. Let's create a new React Native project called "AwesomeProject":
+
+```shell
+npx react-native init AwesomeProject
```

Comment:
Question: now that 0.68 is out, does this command need to use a particular template? Is it sufficient to just add a note saying "Use 0.68 or later"?

Reply:
By default, this creates an app using the template shipped with the `0.68`. 
I will specify that we can install different versions of RN using the `--version` switch and that the New Architecture is supported starting from version `0.68`.

What do you think?

Reply:
That sounds good to me!